test/test_common/utility.h

```diff
@@ -1040,7 +1040,9 @@ makeHeaderMap(const std::initializer_list<std::pair<std::string, std::string>>&
 
 namespace Api {
 ApiPtr createApiForTest();
+ApiPtr createApiForTest(Random::RandomGenerator& random);
 ApiPtr createApiForTest(Stats::Store& stat_store);
+ApiPtr createApiForTest(Stats::Store& stat_store, Random::RandomGenerator& random);
```

Comment:
this is only continuing a pattern that can be blamed on me, but I wonder if we should now convert to cxx17 call-style here, e.g. https://stackoverflow.com/questions/6181715/convenient-c-struct-initialisation

Reply:
Does Envoy style allow designated initializers? They're part of the C++20 spec and the Google style guide has a special exemption for them.